Project/Wikode/wikodeApp/utils/articleManager.py

```diff
@@ -96,6 +96,14 @@ def getAuthors(self):
                             'ForeName': author.get('ForeName'),
                             'Initials': author.get('Initials')
                         }
+
+                        identifier = author.get('Identifier')
+                        if identifier is not None:
```

Comment:
Double negative if clause olmus.

`if identifier is not None:` 
yerine
`if identifier` yapilabilir.